internal/core/items/text.rs

```diff
@@ -380,22 +380,26 @@ impl Item for TextInput {
 
         match event.event_type {
             KeyEventType::KeyPressed => {
+                match event.text_shortcut() {
+                    Some(text_shortcut) => match text_shortcut {
+                        TextShortcut::Move(direction) => {
+                            TextInput::move_cursor(self, direction, event.modifiers.into(), window);
+                            return KeyEventResult::EventAccepted;
+                        }
+                        TextShortcut::DeleteForward => {
+                            TextInput::delete_char(self, window);
+                            return KeyEventResult::EventAccepted;
+                        }
+                        TextShortcut::DeleteBackward => {
+                            TextInput::delete_previous(self, window);
+                            return KeyEventResult::EventAccepted;
+                        }
+                    },
+                    None => (),
+                };
```

Comment:
If you want, you can also write this with one match instead of two. At your discretion :-)

```suggestion
                match event.text_shortcut() {
                    Some(TextShortcut::Move(direction)) => {
                        TextInput::move_cursor(self, direction, event.modifiers.into(), window);
                        return KeyEventResult::EventAccepted;
                    }
                    Some(TextShortcut::DeleteForward) => {
                        TextInput::delete_char(self, window);
                        return KeyEventResult::EventAccepted;
                    }
                    Some(TextShortcut::DeleteBackward) => {
                        TextInput::delete_previous(self, window);
                        return KeyEventResult::EventAccepted;
                    }
                    _ => (),
                };
```

Reply:
I had that at first, but changed it to remove all the `Some()`s.
(Initially I had more of them, because I wanted to move some `TextCursorDirection`s into `TextShortcut`, which was a bad idea)
But it will be more `Some()`s again sooner or later, so i prefer it like that.

Reply:
Alright :)